tools/validate-xep0001-conformance.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+# Checks if provided file (filename to be provided as first argument) conforms to XEP-0001.
+# See https://github.com/xsf/xeps/issues/1235
+#
+# Expected to be executed from the directory that holds all xep XML files.
+# Requires one argument: the file name of the xep to be validated, eg:
+#
+# $ tools/validate-xep0001-conformance.sh xep-0010.xml
+#
+# exit status will be non-zero upon validation failure.
+#
+# requires: bash, xmllint
+
+set -e
+
+VALIDATION_RESULT=0
```

Comment:
Matter of taste: I tend to use `UPPER_SNAKE_CASE` only for environment variables which are either passed into the script or exported to called programs later on. For script-local variables, I use `lower_snake_case`.